gef.py

```diff
@@ -3113,10 +3186,7 @@ def get_generic_arch(module, prefix, arch, mode, big_endian, to_string=False):
 
     else:
         arch = getattr(module, "{:s}_ARCH_{:s}".format(prefix, arch))
-        if prefix == "KS" and mode == "ARM":
```

Comment:
Why are we getting rid of this?

Reply:
If you read the comment below, and test again, you can see that now `keystone.KS_MODE_ARM` exists. This check is not only unnecessary but will also fail to correctly build the keystone instance.